packages/serverpod/lib/src/server/server.dart

```diff
@@ -184,8 +176,24 @@ class Server {
     stdout.writeln('$name stopped');
   }
 
+  void _handleRequestWithErrorBoundary(HttpRequest request) async {
+    // [Future.sync] ensures no synchronous error is accidentally thrown from the handler
+    // https://dart.dev/libraries/async/futures-error-handling#solution-using-future-sync-to-wrap-your-code
+    await Future.sync(() {
+      return _handleRequest(request);
```

Comment:
Question: Is no await required here?

Reply:
The callback returns a FutureOr which is passed on to the caller if it is a Future, so the outer await is sufficient.